cms/envs/common.py

```diff
@@ -904,6 +904,7 @@
 
 SOUTH_MIGRATION_MODULES.update({
     'edx_notifications': 'edx_notifications.stores.sql.migrations',
+    'problem_builder': 'problem_builder.south_migrations',
 })
```

Comment:
@e-kolpakov That's how it is done in mentoring - https://github.com/edx-solutions/xblock-mentoring/tree/master/mentoring/migrations so I wouldn't expect something specific to the workbench